addons/payment_adyen/controllers/main.py

```diff
@@ -263,11 +263,14 @@ def adyen_notification(self):
                     notification_data['resultCode'] = 'Authorised' if success else 'Error'
                 else:
                     continue  # Don't handle unsupported event codes and failed events
-
-                # Handle the notification data as a regular feedback
-                PaymentTransaction.sudo()._handle_feedback_data('adyen', notification_data)
-            except ValidationError:  # Acknowledge the notification to avoid getting spammed
-                _logger.exception("unable to handle the notification data; skipping to acknowledge")
+            except ValidationError:
+                _logger.warning("unable to find the transaction; skipping to acknowledge")
```

Comment:
Maybe add a comment to explain why here it's only a warning?

Reply:
Done, but if you have a better idea, let me now!